JitsiConference.js

```diff
@@ -579,22 +579,27 @@ JitsiConference.prototype.removeCommandListener = function(command) {
 /**
  * Sends text message to the other participants in the conference
  * @param message the text message.
+ * @param elementName the element name to encapsulate the message.
  * @deprecated Use 'sendMessage' instead. TODO: this should be private.
  */
-JitsiConference.prototype.sendTextMessage = function(message) {
+JitsiConference.prototype.sendTextMessage = function(
+        message, elementName = 'body') {
     if (this.room) {
-        this.room.sendMessage(message);
+        this.room.sendMessage(message, elementName);
     }
 };
 
 /**
  * Send private text message to another participant of the conference
+ * @param id the id of the participant to send a private message.
  * @param message the text message.
+ * @param elementName the element name to encapsulate the message.
  * @deprecated Use 'sendMessage' instead. TODO: this should be private.
  */
-JitsiConference.prototype.sendPrivateTextMessage = function(id, message) {
+JitsiConference.prototype.sendPrivateTextMessage = function(
+        id, elementName = 'body', message) {
     if (this.room) {
-        this.room.sendPrivateMessage(id, message);
+        this.room.sendPrivateMessage(id, elementName, message);
```

Comment:
The argument order doesn't seem correct (ChatRoom#sendPrivateMessage expects [id, message, elementName]).